examples/sfdx-project/package-complex.json

```diff
@@ -11,7 +11,8 @@
       "versionName": "Winter ‘20",
       "versionDescription": "Welcome to Winter 2020 Release of Expense Manager Util Package",
       "versionNumber": "4.7.0.NEXT",
-      "definitionFile": "config/scratch-org-def.json"
+      "definitionFile": "config/scratch-org-def.json",
+      "functions": ["functions/myfunction", "funcitons/anotherfunction"]
```

Comment:
@cwallsfdc - is this typo intentional "funcitons" ?

Reply:
Ah, good catch.  Fixing...